x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -15893,19 +15893,14 @@
     "xpack.securitySolution.endpoint.resolver.panel.error.goBack": "このリンクをクリックすると、すべてのプロセスのリストに戻ります。",
     "xpack.securitySolution.endpoint.resolver.panel.processDescList.events": "イベント",
     "xpack.securitySolution.endpoint.resolver.panel.processEventCounts.events": "イベント",
-    "xpack.securitySolution.endpoint.resolver.panel.processEventListByType.eventDescriptiveName": "{descriptor} {subject}",
```

Comment:
:information_source: I still have no idea how to do this

Reply:
To remove them, you can do a find all on the `id` and just delete, but are there some additional changes we have to do if we change the text for a specific field/translation?